packages/awslint/lib/rules/durations.ts

```diff
@@ -3,6 +3,7 @@ import { Linter } from '../linter';
 
 const DURATION_FQN = '@aws-cdk/core.Duration';
 const DURATION_SUFFIX = /(Days|Milli(?:(?:S|s)econd)?s?|Sec(?:ond)?s?)$/;
+const EXCLUDE_ANNOTATION_DURATION_PROP_TYPE = '[disable-awslint:duration-prop-type]';
```

Comment:
Had to create this to avoid those errors for all props interfaces that extend `TaskStateBaseProps`. I think this is a better solution than adding all the exclusions in the `package.json` in `aws-stepfunctions-tasks`.

<details>
  <summary>Build errors <code>awslint:duration-prop-type</code></summary>

```
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.AthenaGetQueryExecutionProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.AthenaGetQueryExecutionProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.AthenaGetQueryResultsProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.AthenaGetQueryResultsProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.AthenaStartQueryExecutionProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.AthenaStartQueryExecutionProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.AthenaStopQueryExecutionProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.AthenaStopQueryExecutionProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.BatchSubmitJobProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.BatchSubmitJobProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CallApiGatewayEndpointBaseProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CallApiGatewayEndpointBaseProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CallApiGatewayHttpApiEndpointProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CallApiGatewayHttpApiEndpointProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CallApiGatewayRestApiEndpointProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CallApiGatewayRestApiEndpointProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CallAwsServiceProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CallAwsServiceProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CodeBuildStartBuildProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.CodeBuildStartBuildProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.DynamoDeleteItemProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.DynamoDeleteItemProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.DynamoGetItemProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.DynamoGetItemProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.DynamoPutItemProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.DynamoPutItemProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.DynamoUpdateItemProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.DynamoUpdateItemProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EcsRunTaskProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EcsRunTaskProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EksCallProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EksCallProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrAddStepProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrAddStepProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrCancelStepProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrCancelStepProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrContainersCreateVirtualClusterProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrContainersCreateVirtualClusterProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrContainersDeleteVirtualClusterProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrContainersDeleteVirtualClusterProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrContainersStartJobRunProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrContainersStartJobRunProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrCreateClusterProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrCreateClusterProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrModifyInstanceFleetByNameProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrModifyInstanceFleetByNameProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrModifyInstanceGroupByNameProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrModifyInstanceGroupByNameProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrSetClusterTerminationProtectionProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrSetClusterTerminationProtectionProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrTerminateClusterProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EmrTerminateClusterProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EvaluateExpressionProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EvaluateExpressionProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EventBridgePutEventsProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.EventBridgePutEventsProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.GlueDataBrewStartJobRunProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.GlueDataBrewStartJobRunProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.GlueStartJobRunProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.GlueStartJobRunProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.LambdaInvokeProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.LambdaInvokeProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateEndpointConfigProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateEndpointConfigProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateEndpointProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateEndpointProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateModelProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateModelProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateTrainingJobProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateTrainingJobProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateTransformJobProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerCreateTransformJobProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerUpdateEndpointProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SageMakerUpdateEndpointProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SnsPublishProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SnsPublishProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SqsSendMessageProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.SqsSendMessageProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.StepFunctionsInvokeActivityProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.StepFunctionsInvokeActivityProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.StepFunctionsStartExecutionProps.heartbeatTimeout] property must be typed @aws-cdk/core.Duration 
@aws-cdk/aws-stepfunctions-tasks: error: [awslint:duration-prop-type:@aws-cdk/aws-stepfunctions-tasks.StepFunctionsStartExecutionProps.taskTimeout] property must be typed @aws-cdk/core.Duration 
```
</details>